.evergreen/config.yml

```diff
@@ -2504,6 +2507,14 @@ axes:
           test_encryption: true
           test_crypt_shared: true
         batchtime: 10080  # 7 days
+      - id: "encryption_crypt_shared_old_server"
+        display_name: "Encryption shared lib old server versions"
+        tags: ["encryption_tag"]
+        variables:
+          test_encryption: true
+          test_crypt_shared: true
+          test_crypt_shared_latest: true
+        batchtime: 10080  # 7 days
```

Comment:
What do you think about testing this combination (test_crypt_shared_latest with <6.0 servers) in the encryption_crypt_shared axis instead of adding new variants? This would reduce the visual noise in the EVG UI.

Reply:
I initially was going to do that, but the spec states that we must use the latest version of crypt shared for older server versions. Simply removing those excluded server versions attempts to download the version of crypt shared corresponding to that server version, which fails when <6.0. I thought this would be more clear than adding specific logic in run-tests.sh to use latest with older server versions.

Reply:
I think it's still worthwhile in the long run so that we can avoid duplicating the "Encryption shared lib" variants.

Reply:
That is fair. I will add a note in config.yml explaining that it has this new behavior.